app/controllers/applications_controller.rb

```diff
@@ -0,0 +1,19 @@
+class ApplicationsController < ApplicationController
+  def new
+    @favorites = Pet.find(cookies[:favorites].split(','))
+  end
+
+  def create
+    favorites = cookies[:favorites].split(',')
+
+    # possible refactor to helper method
+    params[:favorite_ids].each do |favorite_id|
+      favorites.delete(favorite_id)
+    end
+
+    cookies[:favorites] = favorites.join(',')
```

Comment:
(Leaving a note for future reference: we discussed moving this later to a PORO to remove code related to cookie manipulation from controller